src/components/Toggle/index.js

```diff
@@ -0,0 +1,36 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = ['active', 'inactive'];
+
+const html = `
+  <div class="toggle-container">
+    <input type="checkbox" id="toggle-input" class="toggle-container__input" data-select="toggle-input">
+    <label for="toggle-input" class="toggle-container__label" data-select="toggle-label"></label>
```

Comment:
O uso dos atributos "id" e do "for" vai dar problema.
Se criarmos duas instancias do toggle o id já não será único.
Temos um utilitário para resolver isso.

Importe o utilitário e crie um gerador de ids
```js
import { Component, createIDFactory } from 'pet-dex-utilities';
const generateID = createIDFactory("toggle");
```

Depois você utiliza ele dentro do componente
```js
const id = generateID();
this.selected.get('toggle-input').setAttribute('id', id);
this.selected.get('toggle-label').setAttribute('for', id);
```

Reply:
Ficou faltando remover os ids da template string na variavel html

Reply:
Está faltando remover os IDs na template string do html